lagunita/lms/static/sass/partials/base/_overrides.scss

```diff
@@ -31,4 +31,8 @@ $footer-logo-width: 100px;
     display: none;
 }
 
+$screen_break_xs_s: 500px;
```

Comment:
Do we have these defined elsewhere too?

Reply:
Also, why do we list a range of sizes instead of just one?

Why not `small, medium, large, xl` instead of `xs -> small, small -> medium, etc...`?

Reply:
cool.